ui/widgets/nuxeo-operation-button.js

```diff
@@ -198,7 +198,7 @@ import '../actions/nuxeo-action-button-styles.js';
           );
 
           if (this.download) {
-            return this._download(response);
+            this._download(response);
```

Comment:
Perhaps we should keep the return, otherwise errors thrown inside `_download` won't be caught in the catch on line 204. Otherwise, LGTM.

Reply:
Errors in `_download` are thrown as exceptions so removal of the return should have no impact here.
I removed the return just to try and make it more explicit that we're loosing control of the download it self, all we do is create a link and click it which is synchronous. We have no way to track progress on that download anymore...

Reply:
Not sure if I completely understood what @Gabez0r was saying, but I would say that we don't need the return.

Reply:
So, if a `missing Content-Disposition header` is thrown, are we ok with just not displaying it on toast? I'd expect this not to be the case, since we've have tickets complaining of lack of feedback on uploads/downloads fail for whatever reason.

Reply:
Sorry,  `_download` no longer runs async code, so it's ok! 👍